lib/crowi/index.js

```diff
@@ -107,12 +111,42 @@ Crowi.prototype.isPageId = function(pageId) {
   return false
 }
 
+Crowi.prototype.beforeSetConfig = function(config) {
```

Comment:
[IMO] How about `publishConfig` for name?
I couldn't understand that this function is used for sharing new configuration to another nodes from `beforeSetConfig` name.